pkg/triggeruninstall/triggeruninstall.go

```diff
@@ -77,35 +81,47 @@ func TriggerUninstall(
 		default:
 		}
 
-		configPolicies, err := dynamicClient.Resource(configPolicyGVR).Namespace(policyNamespace).List(
-			ctx, metav1.ListOptions{},
-		)
-		if err != nil {
-			if k8serrors.IsServerTimeout(err) || k8serrors.IsTimeout(err) {
-				klog.Infof("Retrying listing the ConfigurationPolicy objects due to error: %s", err)
+		cleanedUp := true
 
-				continue
+		for _, ns := range policyNamespaces {
+			policyClient := dynamicClient.Resource(configPolicyGVR).Namespace(ns)
+
+			configPolicies, err := policyClient.List(ctx, metav1.ListOptions{})
+			if err != nil {
+				if k8serrors.IsServerTimeout(err) || k8serrors.IsTimeout(err) {
+					klog.Infof("Retrying listing the ConfigurationPolicy objects due to error: %s", err)
```

Comment:
Is this retrying? Since it uses continue, it skips the current namespace in this iteration."

Reply:
Ah, good catch, I need to make it `continue` the bigger loop I think

Reply:
Or maybe just mark `cleanedUp = false`... but now that I'm looking at it closer, I don't quite follow why we're specially handling these specific errors. @mprahl do you remember?